frb_codegen/src/others.rs

```diff
@@ -35,8 +35,12 @@ lazy_static! {
         vec!["store_dart_post_cobject".to_string()];
 }
 
-pub const CODE_HEADER: &str = "// AUTO GENERATED FILE, DO NOT EDIT.
-// Generated by `flutter_rust_bridge`.";
+const CODE_HEADER: &str = "// AUTO GENERATED FILE, DO NOT EDIT.
```

Comment:
Sure. Indeed I prefer something like:

```
const CODE_HEADER_FORMAT: &str = "// AUTO GENERATED FILE, DO NOT EDIT.
// Generated by `flutter_rust_bridge`@ {}.";

pub fn code_header() -> String {
    format!(CODE_HEADER_FORMAT, env!("CARGO_PKG_VERSION"))
}
```

So that we can freely place format anywhere. But anyway this is just nits

Reply:
<img width="758" alt="Screen Shot 2565-09-05 at 15 11 15" src="https://user-images.githubusercontent.com/21016014/188400905-fe57f9a2-eaef-4def-a7c4-9fd45105cc39.png">
looks like it needs to be explicit, what do you think ?

Reply:
I wondered, so I also tried with raw string literals, to no avail:

<img width="759" alt="Screen Shot 2565-09-05 at 15 15 48" src="https://user-images.githubusercontent.com/21016014/188401886-6c287c08-9c34-4ae9-a402-469ece5b73a0.png">

<img width="755" alt="Screen Shot 2565-09-05 at 15 15 29" src="https://user-images.githubusercontent.com/21016014/188401876-e08a9cfb-215c-4ed1-9b48-22ce8937ba55.png">

Reply:
Ah you are right. This is because `format!` is a macro not a function. Then what about directly inline it?

```
pub fn code_header() -> String {
    format!("// AUTO GENERATED FILE, DO NOT EDIT.
// Generated by `flutter_rust_bridge`@ {}.", 
      env!("CARGO_PKG_VERSION")
  )
}

Reply:
indeed, done in 11240c12b1c53138931112a69813e477f1c0881f 👌